edx-platform/pearson-vue-theme/lms/student-account/register.underscore

```diff
@@ -1,69 +1,70 @@
-<div class="js-form-feedback" aria-live="assertive" tabindex="-1">
+<div class="background-login"></div>
+
+<div>
+  <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
 </div>
 
-<% if (!context.syncLearnerProfileData) { %>
-	<div class="toggle-form">
-		<span class="text"><%- edx.StringUtils.interpolate(gettext('Already have an {platformName} account?'), {platformName: context.platformName }) %></span>
-		<a href="#login" class="form-toggle" data-type="login"><%- gettext("Sign in.") %></a>
-	</div>
-<% } %>
+<div class="signin-container">
+  <div class="js-form-feedback" aria-live="assertive" tabindex="-1"></div>
 
-<form id="register" class="register-form" autocomplete="off" tabindex="-1" method="POST">
 
-    <% if (!context.currentProvider) { %>
-        <% if (context.providers.length > 0 || context.hasSecondaryProviders) { %>
-            <div class="login-providers">
-                <div class="section-title lines">
-                    <h3>
-                        <span class="text"><%- gettext("Create an account using") %></span>
-                    </h3>
-                </div>
-                <%
-                _.each( context.providers, function( provider) {
-                    if ( provider.registerUrl ) { %>
-                        <button type="button" class="button button-primary button-<%- provider.id %> login-provider register-<%- provider.id %>" data-provider-url="<%- provider.registerUrl %>">
-                            <div class="icon <% if ( provider.iconClass ) { %>fa <%- provider.iconClass %><% } %>" aria-hidden="true">
-                                <% if ( provider.iconImage ) { %>
-                                    <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
-                                <% } %>
-                            </div>
-                            <span aria-hidden="true"><%- provider.name %></span>
-                            <span class="sr"><%- _.sprintf( gettext("Create account using %(providerName)s."), {providerName: provider.name} ) %></span>
-                        </button>
-                <%  }
-                }); %>
+  <div class="toggle-form">
+    <span class="text"><%- gettext("First time here?") %></span>
+    <a href="#login" class="form-toggle" data-type="register"><%- gettext("Create an account.") %></a>
```

Comment:
Is this a login or a register hyperlink? 
href="#login" 
data-type="register"
text = "Create an account."

Maybe we can change the form id or the hyperlink text in order to keep coherence.
@luismorenolopera

Reply:
this redirects to the registration page